src/timeseriesflattener/testing/load_synth_data.py

```diff
@@ -122,4 +122,4 @@ def load_synth_text(
     Returns:
         pd.DataFrame
     """
-    return load_raw_test_csv("synth_txt_data.csv", n_rows=n_rows)
+    return load_raw_test_csv("synth_text_data_matched.csv", n_rows=n_rows)
```

Comment:
What does matched mean here? :-)

Reply:
Good question! That the values in the entity_id column match those from the synthetic prediction times. Can rename to "synth_text_data.csv" instead and remove the old file